src/equations/compressible_navier_stokes.jl

```diff
@@ -2,10 +2,12 @@
 """
     struct BoundaryConditionNavierStokesWall
 
-Creates a wall-type boundary conditions for the compressible Navier-Stokes equations.
+Creates a wall-type boundary conditions for the compressible Navier-Stokes equations, see
+[`CompressibleEulerEquations1D`](@ref), [`CompressibleEulerEquations2D`](@ref), and
+[`CompressibleEulerEquations3D`](@ref).
```

Comment:
It may be a bit confusing to only link to the Euler equations when talking baout Navier-Stokes.

Reply:
Yeah, that was meant to be the `CompressibleNavierStokesDiffusion1D`etc - thanks!